python/extensions/private/requirements.bzl.tmpl

```diff
@@ -29,7 +33,16 @@ def dist_info_requirement(name):
     return "%%MACRO_TMPL%%".format(_clean_name(name), "dist_info")
 
 def entry_point(pkg, script = None):
-    """entry_point returns the target of the canonical label of the package entrypoints.
+    """entry_point returns the target of the canonical label of the package entry points.
     """
-    # TODO: https://github.com/bazelbuild/rules_python/issues/1262
-    print("not implemented")
+    selects = _entry_point(
+        tmpl = "@@%%NAME%%//{pkg}/bin_py{version_label}:{script}",
+        pkg = _clean_name(pkg),
+        script = script,
+        packages = _packages,
+        default_version = _default_py_version,
+    )
+    if selects == None:
+        fail("Package '{}' does not exist, select one from: {}".format(pkg, _packages.keys()))
+
+    return select(selects)
```

Comment:
```suggestion
    # NOTE: We return a select() expression instead of an alias to such an expression
    # to avoid having to eagerly load all versions of the wheel. See
    # https://github.com/bazelbuild/rules_python/issues/1262 for discussion.
    return select(selects)
```